buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1827,7 +1827,8 @@ class BeamModulePlugin implements Plugin<Project> {
                     configuration: (configuration.shadowClosure ? 'shadow' : 'implementation'), scope: 'compile')
                 generateDependenciesFromConfiguration(configuration: 'provided', scope: 'provided')
 
-                if (!boms.isEmpty()) {
+                // LinkedIn: no boms in pom.xml so it can be used in MPs
```

Comment:
Maybe also mention in the comment that we only need to skip boms for LinkedIn Snapshot build?

Reply:
Makes sense. Added more comments.